### PR TITLE
LPS-103098

### PR DESCRIPTION
**Pull request by ChrisKian**
LPS-103098

Please let me know if you have any questions, thanks!

### CODE REVIEW

**Comment by ChrisKian** [comment]
ci:test:sf

**Comment by ChrisKian** [comment]
ci:test:relevant

**Comment by yichenroy** [comment]
@hhuijser looks like it's failing on the build-profile-dxp related task:
> /opt/dev/projects/github/liferay-portal/build.xml:1050: The following error occurred while executing this line:
Target "prepare-profile-dxp-properties" does not exist in the project "null".

**Comment by drewbrokke** [comment]
@ChrisKian @robertoDiaz Thanks for sending this, I like the change. Can you please re-send this with regression test cases? Besides that I think it's fine.

**Comment by hhuijser** [comment]
> @hhuijser looks like it's failing on the build-profile-dxp related task:
> 
> > /opt/dev/projects/github/liferay-portal/build.xml:1050: The following error occurred while executing this line:
> > Target "prepare-profile-dxp-properties" does not exist in the project "null".

Looking into that

**Comment by hhuijser** [comment]
ci:test:relevant

**Comment by robertoDiaz** [comment]
ci:test:relevant

File: portal-impl/src/com/liferay/portal/service/impl/UserLocalServiceImpl.java [hunk 1]
```diff
@@ -4089,23 +4086,15 @@ public void unsetRoleUsers(long roleId, List<User> users)
...
-		String roleName = role.getName();
-
-		if ((roleName.equals(RoleConstants.ADMINISTRATOR) &&
-			 (getRoleUsersCount(role.getRoleId()) <= 1))) {
+		List<User> users = new ArrayList<>(userIds.length);
 
-			throw new RequiredRoleException.RequiredAdminRoleException();
-		}
+		for (Long userId : userIds) {
+			User user = userLocalService.fetchUser(userId);
```
Comment (robertoDiaz) [inline portal-impl/src/com/liferay/portal/service/impl/UserLocalServiceImpl.java]: hey @ChrisKian 

fetchMethod could return a null object, we should use a get (or do a null check)

**Comment by robertoDiaz** [comment]
Just started reviewing :)

**Pull request closed without merging**